### DISCUSSION TITLE
Run list and delete orphaned entities action results in growing list

### DISCUSSION THREAD

**Discussion by swa72** [frenck/spook#815] [Q&A]
I would assume that once I call `homeassistant.list_orphaned_database_entities` and then `homeassistant.delete_all_orphaned_entities` that the action removes orphaned entities. 

Can I further assume that another `homeassistant.list_orphaned_database_entities` then provides an empty list? In my case, even more entities show up in the list of the second call. Odd? 

Or do I have to purge those entities from the recorder as well (documentation is not quite clear on that)?

**Comment by frenck** [comment]
> I would assume that once I call homeassistant.list_orphaned_database_entities and then homeassistant.delete_all_orphaned_entities that the action removes orphaned entities.

That isn't correct. One mentions the database, the other doesn't.

../Frenck
